src/components/QuizGame/index.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import { fetchQuizQuestions } from "../../components/QuizGameApi";
+import QuestionCard from "../../components/QuizGameQuestions";
+import "./index.css";
+const Difficulty = {
+  EASY: "easy",
+  MEDIUM: "medium",
+  HARD: "hard",
+};
+const TOTAL_QUESTIONS = 10;
+const QuizGame = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [questions, setQuestions] = useState([]);
+  const [currentQuestionNumber, setCurrentQuestionNumber] = useState(0);
+  const [userAnswers, setUserAnswers] = useState([]);
+  const [score, setScore] = useState(0);
+  const [isGameOver, setIsGameOver] = useState(false);
+  const [isGameStart,setIsGameStart]=useState(false)
+  const startQuiz = async () => {
+    setIsLoading(true);
+    setIsGameOver(false);
+    setIsGameStart(true)
+    const newQuestions = await fetchQuizQuestions(
+      TOTAL_QUESTIONS,
+      Difficulty.EASY
+    );
```

Comment:
If this game is coded to show only easy questions there is no need for `difficulty `object in line 5

Reply:
thats true, but i implement it anyway if you want to change the difficulty you can change it easily, if you see there is no need for it i can delete the obj.